### ISSUE TITLE
Change the color of the cell based if condition

### ISSUE THREAD

**Issue by gudipudipradeep** [ruilisi/fortune-sheet#186]
conditional cell color change.  
if(current_cel > other_cel: green, red)

**Comment by shuanhukexuejia** [comment]
Is there any example in excel or google sheet

**Comment by gudipudipradeep** [comment]
@shuanhukexuejia  not sure, let me check

**Comment by gudipudipradeep** [comment]
@shuanhukexuejia Conditional Formatting in excel.
https://www.ablebits.com/office-addins-blog/change-background-color-excel-based-on-cell-value/

**Comment by shuanhukexuejia** [comment]
@gudipudipradeep There is some different from the feature you discribe, conditional formatting is ``` cellValue >  specificValue``` while what you desire is ``` cellValue > otherCellValue ```

**Comment by gudipudipradeep** [comment]
@shuanhukexuejia that's correct, Just  to know how we can achieve this kind of behavior here.

**Comment by shuanhukexuejia** [comment]
> @shuanhukexuejia that's correct, Just to know how we can achieve this kind of behavior here.

Maybe it should base on conditional formatting(still incomplete), then we can modify it to make it compatible with custom formatting rules based on cellValue
